### DISCUSSION TITLE
fichier de configuration fenetres

### DISCUSSION THREAD

**Discussion by rvtt51** [mytourbook/mytourbook#1080] [General]
Bonjour,
Est-il posible de connaitre le nom du fichier de configuration qui permet d'agencer les fenêtres de MTB : Graphique des parcours, Livre des parcours, Carte 2D, etc ... 
Ceci pour installer MTB sur mon ordi portable et éviter de refaire la config des fenêtres. Merci.
Bien sportivement.
RV(tt)

**Comment by FJBDev** [comment]
Les fichiers de configurations se trouvent dans le repertoire `.metadata`. Plus d'informations se trouvent dans [la documentation](https://mytourbook.sourceforge.io/mytourbook/index.php/documentation/backup)

![image](https://user-images.githubusercontent.com/25235665/234873687-451e3857-9b92-4577-87f6-73e2e332c238.png)
